src/main/webapp/app/overview/course-overview.component.html

```diff
@@ -103,6 +108,64 @@
                 </div>
             </mat-sidenav>
             <mat-sidenav-content [ngClass]="{ 'sidenav-height-dev': !isProduction || isTestServer }">
+                <!-- Dropdown menu -->
+                <div class="dropdown">
+                    <div class="dropdown-content" [hidden]="!dropdownOpen" [style.top]="dropdownOffsetToString()">
```

Comment:
We typically want to avoid function calls in the template - can you try `[style.top.px]="dropdownOffset"`?